### PR TITLE
New package: AdGuard.AdGuard.Beta version 7.17.4622.0

### PR DESCRIPTION
**Pull request by rakleed**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140762)

### CODE REVIEW

**Comment by rakleed** [comment]
Link could be found at https://adguard.com/en/beta.html?platform=windows

File: manifests/a/AdGuard/AdGuard/Beta/7.17.4622.0/AdGuard.AdGuard.Beta.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: AdGuard.AdGuard.Beta
+PackageVersion: 7.17.4622.0
+PackageLocale: en-US
+Publisher: Adguard Software Limited
+PackageName: AdGuard
+PackageUrl: https://adguard.com
```
Comment (stephengillie) [inline manifests/a/AdGuard/AdGuard/Beta/7.17.4622.0/AdGuard.AdGuard.Beta.locale.en-US.yaml:9]:
```suggestion
PackageUrl: https://adguard.com/en/beta.html
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @rakleed,

This PackageUrl points at the beta version website.

**Comment by rakleed** [comment]
@stephengillie

**Comment by rakleed** [comment]
For some reason wingetbot is ignoring this command for the second time ([the first time](https://github.com/microsoft/winget-pkgs/pull/140762#issuecomment-1962196914)).

**Comment by stephengillie** [comment]
Sorry for its seemingly fickle behavior lately.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**